hirwaf/teamwork
# #168348993 Setting Up API environment

#### What does this PR do?
Setting up API environment
#### Description of Task to be completed?
- [x] **Having these installed**
- NodeJS
- Express
- [x] **Having these packages installed & configured**
- bcrypt
- cors
- dotenv
- moment
- morgan
- swagger-ui-express
- babel
- chai
- coveralls
- eslint
- mocha
- nyc
- [x] Having API prefix "/api/v1" working
- [x] Having Travis-ci, coveralls,  test configured
- [x] Adding budges to the README.md file
#### How should this be manually tested?
- clone the repo and cd into the project
- run ls -al in the terminal should get a list of all files
- run npm install
- run npm start
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168348993
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
